raphtory-graphql/src/model/mod.rs

```diff
@@ -222,6 +222,27 @@ impl Mut {
         data.insert_graph(path, g).await?;
         Ok(path.to_owned())
     }
+
+    /// Create a subgraph out of some existing graph in the server
+    ///
+    /// Returns::
+    ///    name of the new graph
+    async fn create_subgraph<'a>(
+        ctx: &Context<'a>,
+        parent_path: &str,
+        nodes: Vec<String>,
+        new_path: String,
+        overwrite: bool,
+    ) -> Result<String> {
+        let data = ctx.data_unchecked::<Data>();
+        let parent_graph = data.get_graph(parent_path)?.0.graph.materialize()?;
```

Comment:
Shouldn't materialize the parent graph again? Materialising the subgraph is enough, no?

Reply:
oh you are right, I completely missed that one